modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -145,10 +145,12 @@ export default class NestedTabs extends Base {
 			'aria-expanded': 'false',
 		} );
 
-		$activeContent[ settings.hideTabFn ]();
+		$activeContent[ settings.hideTabFn ]( 0, () => this.onHideTabContent( $activeContent ) );
 		$activeContent.attr( 'hidden', 'hidden' );
 	}
 
+	onHideTabContent() {}
```

Comment:
@udidol Should you not rather add the param here so the signature would be the same?